aws/cloudformation/s3_backed_subdomain.yml

```diff
@@ -93,7 +93,7 @@ Resources:
               OriginAccessIdentity: ""
         ViewerCertificate:
           AcmCertificateArn: !Ref CertificateArn
-          MinimumProtocolVersion: TLSv1
+          MinimumProtocolVersion: TLSv1.2_2021
```

Comment:
This template won't deploy anywhere automatically, but fixing it does ensure future stacks that use it will be correct. We'll just need to manually update the related stacks. AWS changed S3 defaults, so this template is actually broken. I've been poking at an update here:

https://github.com/code-dot-org/code-dot-org/compare/staging...darin/static-site